packages/3id-did-resolver/CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+# [1.2.0-rc.0](/compare/@ceramicnetwork/3id-did-resolver@1.1.0...@ceramicnetwork/3id-did-resolver@1.2.0-rc.0) (2021-03-29)
```

Comment:
Sorry, release candidate version bump snuck in while trying to debug some test failures due to what appeared to be circular dependency issues